testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/grpc/GrpcToJaxrsTest.java

```diff
@@ -0,0 +1,805 @@
+package org.jboss.resteasy.test.grpc;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.logging.Logger;
+import org.jboss.resteasy.utils.PortProviderUtil;
+import org.jboss.resteasy.utils.TestUtil;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.exporter.ZipExporter;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.google.protobuf.Any;
+
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import io.grpc.StatusRuntimeException;
+import jakarta.ws.rs.client.Client;
+import jakarta.ws.rs.client.ClientBuilder;
+import jakarta.ws.rs.core.Response;
+import jaxrs.example.CC1ServiceGrpc;
+import jaxrs.example.CC1_proto;
+import jaxrs.example.CC1ServiceGrpc.CC1ServiceBlockingStub;
+import jaxrs.example.CC1_proto.FormMap;
+import jaxrs.example.CC1_proto.FormValues;
+import jaxrs.example.CC1_proto.GeneralEntityMessage;
+import jaxrs.example.CC1_proto.GeneralReturnMessage;
+import jaxrs.example.CC1_proto.ServletInfo;
+import jaxrs.example.CC1_proto.gCookie;
+import jaxrs.example.CC1_proto.gHeader;
+import jaxrs.example.CC1_proto.gString;
+import jaxrs.example.CC1_proto.org_jboss_resteasy_example___CC2;
+import jaxrs.example.CC1_proto.org_jboss_resteasy_example___CC3;
+import jaxrs.example.CC1_proto.org_jboss_resteasy_example___CC4;
+import jaxrs.example.CC1_proto.org_jboss_resteasy_example___CC5;
```

Comment:
We cannot use package names like this.

Reply:
They come from the example project and its corresponding generated project. I'll changed them.